webapp/tests/forms/steps/test_eligibility_steps.py

```diff
@@ -175,6 +176,36 @@ def test_if_invalid_and_valid_and_second_valid_are_in_prev_step_then_set_prev_ur
             self.assertEqual(previous_url, step.render_info.prev_url)
 
 
+class TestEligibilityStartDisplaySteuerlotseStep(unittest.TestCase):
```

Comment:
Personally, I would add one additional tests for a totally empty session_data and one for empty eligibility data. Nevertheless, this looks good to me.

Reply:
Added them. Is that what you had in mind?

Reply:
Yep